### DISCUSSION TITLE
Redhat new Class room environment

### DISCUSSION THREAD

**Discussion by thinkerpiash** [rdbreak/rhcsa8env#76] [Q&A]
Can you make something like that?

**Comment by Da-Geek** [comment]
> ![RH124-Classroom-Architecture]) Can you make something like that?

Bit of friendly advice @thinkerpiash 

I am quiet confident you have just broken Copyright Law by taking this Image from the Official Red Hat Training,
So you may want to remove this from a Publicly Viewable Website, that also has your Username attached to it.
